pkg/gofr/gofr.go

```diff
@@ -224,6 +237,33 @@ func (a *App) Run() {
 	wg.Wait()
 }
 
+func (a *App) pingGoFr(client *http.Client, s string) {
```

Comment:
Here are some suggestions for a better name

```suggestion
func (a *App) handleTelemetry(client *http.Client, s string) {
```

Or


```suggestion
func (a *App) sendTelemetry(client *http.Client, s string) {
```